### PR TITLE
Add contributing guide

### LINKED ISSUE #3: Write CONTRIBUTING.md for site developers
- write how to setup intellisence for site config
- write how create new pages

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
closes #3

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Contributing guide
+
+## Prerequisites
+
+The following Visual Studio Code extensions are required for IntelliSense in
+site config file:
```
Comment (kbdharun) [inline CONTRIBUTING.md:6]:
```suggestion
site config file:-
```

Comment (EmilySeville7cfg) [reply CONTRIBUTING.md:6]: Is it really necessary? I am confused, about `:-` 😄

Comment (kbdharun) [reply CONTRIBUTING.md:6]: > I am confused, about `:-`, I see such syntax for the first time. 😄

Oh, Now I have a question are the pages written in American or British English?

Comment (kbdharun) [reply CONTRIBUTING.md:6]: > Let's prefer American one. (I didn't think about what language dialect is used.)

Fine then it is GTG.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**